smtk/extension/qt/qtUIManager.h

```diff
@@ -48,15 +48,18 @@ namespace smtk
     Q_OBJECT
 
     public:
-    qtUIManager(smtk::attribute::System &system,
-                const std::string &toplevelViewName);
+    qtUIManager(smtk::attribute::System &system);
       virtual ~qtUIManager();
 
       void initializeUI(QWidget* pWidget, bool useInternalFileBrowser=false);
-      qtBaseView* initializeView(QWidget* pWidget, smtk::common::ViewPtr view,
-        bool useInternalFileBrowser=true);
+      qtBaseView *setSMTKView(smtk::common::ViewPtr v);
```

Comment:
The reason I was forced to call the methods setSMTKView and smtkView was because there was already the topView method as you pointed out.  I can't rename setSMTKView to setTopView because your topView returns a different class (qtBaseView* not ViewPtr) - I think this would be ugly - unless we rename topView to be topQView which also looks bad.  So I would suggest we leave things as is.